tests/yam/agama/auto.pm

```diff
@@ -7,16 +7,25 @@
 use base 'y2_installbase';
 use strict;
 use warnings;
+use serial_terminal;
 
 use testapi 'assert_screen';
 
 
 sub run {
     assert_screen('agama-main-page', 120);
     assert_screen('agama-installing', 60);
+    return 42;
```

Comment:
please try to communicate which code serves to simulate failures to distinguish it from the one to be merged.